@utahdts/utah-design-system/css/6-components/base-components/popups/_modal.scss

```diff
@@ -1,15 +1,66 @@
 @use "../../../1-settings/settings-index";
```

Comment:
- Visual review: There is a dark border around the dialog.
- Is the distance between the title and the content too big?
- Is the distance to the top of the dialog from the top of the Title too big?
- Font weight doesn't match to mockups for the Title (discuss)
- Why does the entire modal get focus? I don't think this is correct behavior. I could be wrong.

- Modals should return the user to the place where the modal was activated. In the case of the sandbox back to the button. We should pass in a ref for the return.
- Is "Forced action" the name of the prop? Should it be? Should it be renamed?

Reply:
Check the document.activeElement to see if you can use that to go back to before the modal renders.

Reply:
Change the name of "Forced action"

Reply:
The border is coming from "dialog" we need to remove it.

Reply:
Use the modal__title to style plain text for the example and not use a heading, I guess.